app/controllers/admin/people_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
```

Comment:
Another commit message comment.

ea7b8d5cbf0173da8af7208d9cec073a20906a47

> Also added a view for the sake of being able to write specs

Remember that each commit is its own thing. I don't necessarily know what came before this so "also" doesn't mean anything to me in this context.

> being able to write specs

This isn't a good enough reason to have a commit. These specs can be written when the actual view is created.